### PR TITLE
Sprint 2

### PR DESCRIPTION
**Pull request by AngelFBI**
# Description
Info in their respectic PR
- #22 
- #24 
- #26 
- #28 
- #30 
- #32 
- #34 
- #36

### CODE REVIEW

File: app/models/list.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class List < ApplicationRecord
+  COLORS = %w[danger success dark light primary secondary warning info].freeze
+  PRIORITIES = %w[High Medium Low].freeze
```
Comment (oskar-raweng) [inline app/models/list.rb:5]: good job using freeze.

File: app/presenters/task_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+class TaskPresenter
+  include ActionView::Helpers
+  include Rails.application.routes.url_helpers
+
```
Comment (oskar-raweng) [inline app/presenters/task_presenter.rb:6]: great job using these modules

**Review by ohenroute — APPROVED** [review]
There are not so many changes in this PR. 

But you did an excellent job.

File: app/models/list.rb (hunk 1 above)
Comment (AngelFBI) [reply app/models/list.rb:5]: Thanks, but rubocop was the one who told me this

File: app/presenters/task_presenter.rb (hunk 2 above)
Comment (AngelFBI) [reply app/presenters/task_presenter.rb:6]: Thanks, I had to read a lot

**Pull request merged**